docs/usage/config-overview.md

```diff
@@ -0,0 +1,209 @@
+# Renovate Configuration Overview
+
+When Renovate runs on a repository, the final config in use is derived from:
+
+- Global config
+- Inherited config
+- Repository config
+- Resolved presets referenced in config
+
+## Types of Config
+
+### Global config
+
+Global config is the term for config defined by the person or team responsible for running the bot.
```

Comment:
```suggestion
Global config is the term for config defined by the person or team responsible for running the bot.
Sometimes it's referred to as "bot config" because it's the config passed to the bot by the person running it.
```